README.md

```diff
@@ -54,8 +54,8 @@ children = [
 
 Pools will be started for each configured `{scheme, host, port}` when Finch is started.
 For any unconfigured `{scheme, host, port}`, the pool will be started the first time
-it is requested. Note pools are not automatically terminated if they are unused, so
-Finch is best suited when you are requesting a known list of static hosts.
+it is requested. Note pools are not automatically terminated by default, if you need to
+terminate them after some idle time, use the `pool_max_idle_time` option.
```

Comment:
We should probably mention that this feature is only supported by HTTP1 pools for now.